src/auth.rs

```diff
@@ -153,6 +161,7 @@ impl Default for AuthStore {
             token: AccessToken::new("".to_string()),
             expiry_time: DateTime::<Utc>::MIN_UTC,
             identity_provider: IdentityProvider::None,
+            refresh_token: RefreshToken::new("".to_string()),
```

Comment:
We may want to store refresh tokens more securely in the future. This was a quick-and-dirty way of doing it at the time, but I'm not sure saving the tokens in plain text on the hard drive is following good security practices...

Reply:
Made an issue here for later https://github.com/8451/arrow-flight-cli/issues/7